lib/rex/post/smb/ui/console/command_dispatcher/shares.rb

```diff
@@ -181,12 +182,31 @@ def cmd_ls(*args)
             print_line table.to_s
           end
 
+          #
+          # Alias the ls command to dir, for those of us who have windows muscle-memory
+          #
+          alias cmd_dir cmd_ls
+
+          def cmd_ls_help
+            print_line 'Usage:'
+            print_line 'ls [options] [glob/path]'
+            print_line
+            print_line 'COMMAND ALIASES:'
+            print_line
+            print_line '    dir'
+            print_line
+            print_line 'Lists contents of directory or file info, searchable'
```

Comment:
Is it searchable? 👀

Reply:
I lifted this from Meterpreter, I'll need to double check that.

Reply:
It is not searchable 😄. Will remove that now.